src/utils/archive.ts

```diff
@@ -3,14 +3,15 @@ import { v4 as uuidv4 } from "uuid";
 import Arweave from "arweave";
 import manifest from "../../public/manifest.json";
 import Transaction from "arweave/node/lib/transaction";
+import { getArDriveTipPercentage, selectTokenHolder } from "./pst";
 
 /**
  * Some basic ArDrive info for transactions
  */
 const ardriveClient = "ArDrive-Web";
 const ardriveVersion = "0.1.0";
 const ArFS = "0.11";
-
+const defaultArDriveMinimumTipAR = 0.00001;
```

Comment:
It's wise to be careful with the decimal places. Default min should be 10M Winston, so consider updating to:
```
const defaultArDriveMinimumTipAR = 0.000_010_000_000;
```

Reply:
[Resolved](https://github.com/th8ta/ArConnect/pull/33/commits/66b11719315a0ac4ea310ba6d165889efd900a59)